nixos/doc/manual/configuration/customizing-packages.section.md

```diff
@@ -12,6 +12,29 @@ Unfortunately, Nixpkgs currently lacks a way to query available
 configuration options.
 :::
 
+::: {.note}
+However, many packages in the `nix` store come with extensions one might add.
+Examples include (arbitrarily selected):
+- [`python39Packages.click`][0]
+- [`vimPlugins.vimacs`][1]
+- [`gnomeExtensions.freon`][2]
+
+These may be leveraged like so:
+```nix
+environment.systemPackages = with pkgs; [
+  foo
+  bar
+  (
+    pass-nodmenu.withExtensions (exts: [
+      exts.pass-audit
+      exts.pass-otp
+    ])
+  )
```

Comment:
```suggestion
  (pass-nodmenu.withExtensions (exts: [
    exts.pass-audit
    exts.pass-otp
  ]))
```
looks better

Reply:
Eh, I don't really care, it's down to personal preference I think? I haven't found an authoritorian style-guide/linter =)

But I trust your experience more than mine! =) Thanks!